src/MulticastService.cs

```diff
@@ -135,9 +135,7 @@ protected static IEnumerable<NetworkInterface> GetNetworkInterfaces()
         {
             return NetworkInterface.GetAllNetworkInterfaces()
                 .Where(nic => nic.OperationalStatus == OperationalStatus.Up)
-                .Where(nic =>
-                    nic.NetworkInterfaceType != NetworkInterfaceType.Loopback &&
-                    nic.NetworkInterfaceType != NetworkInterfaceType.Unknown);
```

Comment:
Why remove check for `Unknown`.  Your previous PR needed it>

Reply:
Well, there is this "utun0" nic on macOS that was causing problems. It was of type "unknown", so I filtered that out.
Now, I do a try catch around the "add membership" as I experienced other scenarios where it would throw. This will also catch and survive the "utun0" nic. So I thought it would be better to remove the "unknown" part, as I honestly do not know what other legit nics out there that are of type "unknown".